CHANGELOG.md

```diff
@@ -74,7 +80,7 @@ All notable changes to this project will be documented in this file.
 - Unified `ClusterRef` type for referring to cluster objects ([#307]).
 
 ### Changed
-- BREAKING: kube 0.66 -> 0.68 ([#303]).
+- BREAKING: kube 0.66 -*Please add a description here. This will become the commit message of the merge request later.*> 0.68 ([#303]).
```

Comment:
Uhhh

Reply:
Oh lol

Reply:
I'm scrolling with my middle mouse button and accidentally pasted this :facepalm: